compiler/compile.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
What's the point of this script? Why can't we take an argument in the OCaml file and fun the compiled version directly?